contracts/src/systems/order/cancel_fungible_order.cairo

```diff
@@ -0,0 +1,258 @@
+#[system]
+mod CancelFungibleOrder {
+    use eternum::components::owner::Owner;
+    use eternum::components::trade::{Trade, Status, TradeStatus};
+
+    use eternum::systems::order::helpers::order;
+
+    use box::BoxTrait;
+    use traits::Into;
+
+    use dojo::world::Context;
+
+    fn execute(ctx: Context, trade_id: u128) {
+
+        let (trade_meta, trade_status) = get!(ctx.world, trade_id, (Trade, Status));
```

Comment:
i like that you use trade_meta instead of meta, can you check if there are other places where we use meta or status instead of trade_meta and trade_status and refactor?